packages/config/config/devices/0x0113/lrm-as.json

```diff
@@ -17,15 +17,23 @@
 	},
 	"associations": {
 		"1": {
-			"label": "Group 1",
+			"label": "Lifeline",
 			"maxNodes": 5,
 			"isLifeline": true
+		},
+		"2": {
+			"label": "Double Tap",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Triple Tap",
+			"maxNodes": 5
 		}
 	},
 	"paramInformation": {
-		"3": {
-			"label": "Night Light",
-			"description": "In night-light mode the LED will turn ON when the switch is turned OFF.",
+		"1": {
+			"label": "Dim Start Level",
+			"description": "Start dimming from that start level or current level",
```

Comment:
This description doesn't make sense. Is it the start level or the current level? Which are you setting by chosing enable?